sdk/loadtestservice/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestAdministrationClient.java

```diff
@@ -967,9 +967,52 @@ public PagedIterable<BinaryData> listLoadTestSearch(RequestOptions requestOption
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @return all test files as paginated response with {@link PagedIterable}.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedIterable<BinaryData> getAllTestFiles(String testId, RequestOptions requestOptions) {
+    PagedIterable<BinaryData> getAllTestFiles(String testId, RequestOptions requestOptions) {
```

Comment:
Was this change handwritten?

Reply:
This is generated. I think Harshan is addressing my comment here that name should follow `list<noun>`.
https://azure.github.io/azure-sdk/java_introduction.html#java-service-client-verbs

Fine for me. Though usually you should sync with other SDKs and check if you should change the swagger directly.